ruled_labels/specs_polkadot.yaml

```diff
@@ -0,0 +1,250 @@
+---
+name: paritytech/polkadot
+description: Import from paritytech/polkadot
+labels:
+  - name: A0-please_review
+    description: Pull request needs code review.
+    color: "666666"
+  - name: A1-needs_burnin
+    description: Pull request needs to be tested on a live validator node before merge. DevOps is notified via matrix
+    color: c7def8
+  - name: A2-insubstantial
+    description: "Pull request requires no code review (e.g., a sub-repository hash update)."
+    color: C6C7F8
+  - name: A3-in_progress
+    description: Pull request is in progress. No review needed at this stage.
+    color: DBC6F8
+  - name: A4-companion
```

Comment:
Does A4-companion remove the need for the "polkadot companion: #X" text in the PR description?

Reply:
AFAIK not, this is just added by the CI if the Cumulus companion check fails.

Reply:
Yeah, because it has no way of knowing which exact PR is a companion unless explicitly specified.